.github/workflows/batch-arm-merge-queue.yml

```
@@ -8,24 +8,40 @@ on:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          for pr in ${{ steps.list-prs.outputs.prs }}; do
+            echo "Arming PR #$pr..."
+            gh pr merge --auto --merge "https://github.com/$GITHUB_REPOSITORY/pull/$pr" --repo "$GITHUB_REPOSITORY"
+          done
+          echo "Done!"
 
       - name: Dry run
-        if: inputs.dry_run == true
+        if: inputs.dry_run == 'true'
         run: echo "DRY RUN: Would arm ${{ steps.list-prs.outputs.count }} PRs"
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)